projects/ngx-charts-on-fhir/src/lib/fhir-chart-summary/summary.service.ts

```diff
@@ -21,9 +21,9 @@ export class ScatterDataPointSummaryService implements SummaryService {
   summarize(layer: DataLayer, range: NumberRange): Record<string, string>[] {
     const current = this.stats.getFormattedStatistics(layer, range);
     const prev = this.stats.getFormattedStatistics(layer, previous(range));
-    const days = Math.ceil((range.max - range.min) / MILLISECONDS_PER_DAY)
+    const days = Math.ceil((range.max - range.min) / MILLISECONDS_PER_DAY);
     const summary = Object.keys(current).map((name) => ({
-      name,
+      [layer.name]: name,
       [`current ${days}${'days'}`]: current[name],
       [`previous ${days}${'days'}`]: prev[name],
```

Comment:
I noticed current and previous headings are missing a space. can you clean this up while you're in here:
```suggestion
      [`current ${days} days`]: current[name],
      [`previous ${days} days`]: prev[name],
```